### PR TITLE
[Indigo] add 2dof torso to cob4-2 including all configuration files

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

**Comment by fmessmer** [comment]
merge together with:
- https://github.com/ipa320/cob_common/pull/149
- https://github.com/ipa320/cob_calibration_data/pull/78

File: cob_hardware_config/cob4-2/urdf/cob4-2.urdf.xacro [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <xacro:include filename="$(find cob_description)/urdf/cob4_base/base.urdf.xacro" />
 
   <!-- torso -->
-  <xacro:include filename="$(find cob_description)/urdf/cob4_torso/torso_static.urdf.xacro" />
+  <xacro:include filename="$(find cob_description)/urdf/cob4_torso/torso_2DOF.urdf.xacro" />
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/urdf/cob4-2.urdf.xacro]: I prefer "torso_2dof"

**Comment by fmessmer** [comment]
see [my comment](https://github.com/ipa320/cob_common/pull/149#issuecomment-86938007)

**Comment by fmessmer** [comment]
I updated this PR - but ipa320/cob_common#150 needs to be fixed first!

**Pull request merged**